csxtools/fastccd/images.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+from ..ext import fastccd
```

Comment:
A pattern mpl and numpy use is to make the c-extension modules 'private' and hide them behind a python layer so that the interface to the c can change while mantaining backward compatibility on the public API.

Also, `ext` tends to refer to source you are vendoring, not your own c-extensions.

Both of those comments are bike shedding.